assets/js/common/Input/InputNumber.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import RcInputNumber from 'rc-input-number';
+
+import classNames from 'classnames';
+
+function InputNumber({
```

Comment:
Hey man, I'm just curious why you went for a component wrapper instead of directly using it?

Reply:
for injecting classes in a more atomic way